pkgs/servers/holochain-go/default.nix

```diff
@@ -0,0 +1,31 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
```

Comment:
Since this was manually edited, we should not claim it was generated.

Reply:
Ah ok that makes sense!